holidays/countries/french_guiana.py

```diff
@@ -0,0 +1,33 @@
+#  python-holidays
+#  ---------------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/dr-prodigy/python-holidays
+#  License: MIT (see LICENSE file)
+
+from holidays.countries.france import FR
+
+
+class FrenchGuiana(FR):
+    # Alias of a FR subdivision that is also officially assigned its own
+    # country code in ISO 3166-1. See
+    # https://en.wikipedia.org/wiki/ISO_3166-2:FR#Subdivisions_included_in_ISO_3166-1
+
+    country = "GF"
+    subdivisions = []
+
+    def _populate(self, year: int) -> None:
```

Comment:
`country` and `subdivisions`? Why?

Reply:
No idea :)

The correct selection should have been the whole populate method (ln. 23-25). My goal was to suggest using init instead of populate as it doesn't seem you're doing actual populate method customization.